grapple/types/rich_text.py

```diff
@@ -0,0 +1,25 @@
+from django.template.loader import render_to_string
+from graphene.types import Scalar
+
+try:
+    from wagtail.rich_text import expand_db_html
+except ImportError:
+    from wagtail.core.rich_text import expand_db_html
+
+from ..settings import grapple_settings
+
+
+class RichText(Scalar):
+    @staticmethod
+    def serialize(rich_text: str):
+        if grapple_settings.RICHTEXT_FORMAT == "html":
+            return render_to_string(
+                "wagtailcore/richtext.html", context={"html": expand_db_html(rich_text)}
+            )
+        return rich_text
+
+    parse_value = str
+
+    @staticmethod
+    def parse_literal(node):
+        return str(node.value)
```

Comment:
I'm not sure what the action is that causes this method to be called. Graphene and graphql-core docs are pretty vague on this and I couldn't trigger through a create or update mutation. Perhaps my lack of knowledge on the implementation of graphql servers.

Reply:
Good thinking, updated in https://github.com/torchbox/wagtail-grapple/pull/266/commits/98ae8c77d0b0d766323833c1bb724f1b81e0cc41